test/functional/rpc_dumptxoutset.py

```diff
@@ -25,7 +25,7 @@ def run_test(self):
 
         FILENAME = 'txoutset.dat'
         out = node.dumptxoutset(FILENAME)
-        expected_path = Path(node.datadir) / 'regtest' / FILENAME
+        expected_path = Path(node.datadir) / self.chain / FILENAME
```

Comment:
This rules out running on mainnet, which doesn't add a chain-named subdir (but I guess that's okay as this test would never be suitable for that)

Reply:
It should work fine for the main chain. Note that self.chain refers to the datadir name, see also https://github.com/bitcoin/bitcoin/pull/17556#discussion_r375189523

```
>>> pathlib.Path('/tmp/a') / '' / 'c'
PosixPath('/tmp/a/c')